.github/workflows/test.yml

```diff
@@ -40,8 +40,8 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
-      - name: Run tests
-        run: bundle exec rake --rakefile Rakefile_ci -T | grep release
+      - name: check that required tasks are available
+        run: bundle exec rake --rakefile Rakefile_ci -T | grep --regexp 'rake strings' --regexp 'rake release' --quiet
```

Comment:
And only now I realize that this will pass if any (not all) the regexp match something…

Reply:
https://github.com/voxpupuli/voxpupuli-release/pull/77 should fix it

Reply:
Looks like we can use something like this (filtering available tasks with regexp for exact match, and check that we had something output as when nothing is found, rake returns 0):

```
bundle exec rake --rakefile Rakefile_ci -T '^strings$' | grep .
bundle exec rake --rakefile Rakefile_ci -T '^release$' | grep .
```